src/libPMacc/include/nvidia/functors/Assign.hpp

```diff
@@ -22,6 +22,8 @@
 
 #pragma once
 
+#include "pmacc_types.hpp"
+
 namespace PMacc
 {
     namespace nvidia
```

Comment:
I agree - we should keep the keep the wrong indentation in mind (I will fix them, as soon as this pull request is merged).